configuration/transitions.md

```diff
@@ -427,3 +428,62 @@ Supported `events_types` are:
     ]
   }
 ```
+
+### self_report
+
+Updates a `data_record` to set its patient to its sender. The resulting doc will have `fields.patient_uuid` and `fields.patient_id` filled with the sender's information. Provides hydrated patient information to subsequent transitions.
+The `sender` is the contact associated with the phone number that sent the original SMS.  
+If a doc already contains a `patient` field, does not have a sender or does not have a configured `form` field, it will be ignored by this transition.
```

Comment:
Consider:

```suggestion
If a doc already contains a `patient` field, does not have a sender or whose `form` is not configured to be enabled for this transition, it will be ignored.
```

When I read my first thought was that you meant a form field that is any configured form (like for the entire application).